common/app-starters-security-common/src/main/java/org/springframework/cloud/stream/app/security/common/SecurityCommonAutoConfigurationProperties.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2015-2016 the original author or authors.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.app.security.common;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
+
+/**
+ * {@code SecurityCommonAutoConfiguration} properties.
+ *
+ * @author Christian Tzolov
+ * @author Artem Bilan
+ */
+@ConfigurationProperties("spring.cloud.security")
```

Comment:
I think would better to name it as `spring.cloud.stream.security`.
Just `spring.cloud` sounds too generic and may be treated as a matter for the whole Spring Cloud.

Although I agree that it is going to work automatically when we place this artifact into classpath any way 😄

Reply:
+1

Reply:
+1